tests/NuGetGallery.Facts/ViewModels/DisplayPackageViewModelFacts.cs

```diff
@@ -924,6 +924,36 @@ public void ReturnsExpectedUser(Package package, User currentUser, string expect
             Assert.Null(versionModel.CustomMessage);
         }
 
+        [Fact]
+        public void VulnerabilitiesDisplayedInOrder()
+        {
+            var package = CreateTestPackage("1.0.0");
+
+            var packageKeyToVulnerabilities = new Dictionary<int, IReadOnlyList<PackageVulnerability>>
+            {
+                { package.Key, new List<PackageVulnerability>
+                    {
+                        new PackageVulnerability { Key = 1, Severity = PackageVulnerabilitySeverity.High },
+                        new PackageVulnerability { Key = 2, Severity = PackageVulnerabilitySeverity.Low },
+                        new PackageVulnerability { Key = 3, Severity = PackageVulnerabilitySeverity.Critical },
+                    }
+                }
+            };
+
+            // Act
+            var model = CreateDisplayPackageViewModel(
+                package,
+                currentUser: null,
+                packageKeyToVulnerabilities: packageKeyToVulnerabilities,
+                readmeHtml: null);
+
+            // Assert
+            var versionModel = model.PackageVersions.Single();
+            Assert.Null(versionModel.CustomMessage);
+            Assert.NotNull(model.Vulnerabilities);
+            Assert.Equal(model.Vulnerabilities, packageKeyToVulnerabilities[package.Key].OrderByDescending(p => p.Severity).ToList().AsReadOnly());
```

Comment:
I'm worried someone could break this logic if they re-order the `PackageVulnerabilitySeverity` enum. To prevent this mistake, we could manually verify the order here:

```suggestion
            Assert.Equal(PackageVulnerabilitySeverity.Critical, model.Vulnerabilities[0].Severity);
            Assert.Equal(PackageVulnerabilitySeverity.High, model.Vulnerabilities[1].Severity);
            Assert.Equal(PackageVulnerabilitySeverity.Low, model.Vulnerabilities[2].Severity);
```

Reply:
For sure :)